lib/src/common/model/repos_releases.dart

```diff
@@ -49,36 +56,35 @@ class Release {
   DateTime publishedAt;
 
   /// The author of this release.
+  @JsonKey(toJson: _authorToJson)
   User author;
 
   /// Release Assets
+  @JsonKey(toJson: _assetsToJson)
   List<ReleaseAsset> assets;
 
-  static Release fromJSON(Map<String, dynamic> input) {
+  static Release fromJson(Map<String, dynamic> input) {
```

Comment:
Since this may be a breaking change, is it possible to leave a `fromJSON` that forwards to this one and mark it deprecated?

Reply:
I'd personally prefer having a major version bump than keeping obsolete stuff.